--- origsrc/cmake-3.0.2/Tests/CMakeLists.txt	2014-09-11 06:24:01.000000000 -0700
+++ src/cmake-3.0.2/Tests/CMakeLists.txt	2014-09-25 18:50:45.130147800 -0700
@@ -2448,6 +2448,7 @@
   set(regex "${regex}|Error message was: ")
   set(regex "${regex}([Cc]ould *n.t resolve host")
   set(regex "${regex}|[Cc]ould *n.t connect to host")
+  set(regex "${regex}|Failed *t*o* connect to")
   set(regex "${regex}|Empty reply from server")
   set(regex "${regex}|The requested URL returned error")
   set(regex "${regex}|libcurl was built with SSL disabled. https: not supported)")
